superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.tsx

```diff
@@ -624,6 +625,12 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
                 />
                 {renderPreferredSelector()}
                 {renderAvailableSelector()}
+                <Alert
```

Comment:
Remove this @AAfghahi already has this set in on the feature branch

Reply:
Oh awesome! Removed in [`this commit`](https://github.com/apache/superset/pull/15119/commits/4cb8935b0b03aec55b4c5d699eafcd4022906340).